javascript/example/DRACOLoader.js

```diff
@@ -15,6 +15,7 @@
 'use strict';
 
 THREE.DRACOLoader = function(manager) {
+    this.decoder = THREE.DRACOLoader.getDecoder();
```

Comment:
We currently have some internal changes that we would like to push to our git first to get them tested before they are integrated to the official THREE.js repo, but in general I think we should be able to fully migrate the loader to THREE.js repo pretty soon.